package.json

```diff
@@ -11,7 +11,8 @@
     "clean": "rimraf build",
     "lint": "eslint --ext js,jsx src test",
     "check": "npm run lint && npm run test:all",
-    "build": "babel src --out-dir build",
+    "build": "babel src --out-dir build && npm run build-global",
+    "build-global": "browserify -e build/index.js -s enzyme -x react/addons -x react/lib/ReactContext -x react/lib/ExecutionEnvironment -o build/global.js",
```

Comment:
Could we update this PR to add this command to the readme instead of adding the script?